recipes/efsw/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
+
+
+required_conan_version = ">=1.53.0"
```

Comment:
```suggestion
required_conan_version = ">=2.1"
```

Implements is available since 2.0.9